### DISCUSSION TITLE
Purple text and line highlighting

### DISCUSSION THREAD

**Discussion by KardashIvan** [FrSkyRC/ETHOS-Feedback-Community#3234] [Q&A]
Hello! After updating to the latest version of my FrSky x20s transmitter, I got an ugly purple screen and badly drawn fonts. 
Repeated changes to the software did not help((( Can you help me? I'm desperate.

**Comment by KardashIvan** [comment]
Here is a link to the video

https://drive.google.com/file/d/1XWmLtlADMDxyVqPjqaZw8NosCVssXmVt/view?usp=sharing

**Comment by bsongis-frsky** [comment]
Hardware issue, fixed now!
